### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by kathirr007** [mustafacagri/mevn-boilerplate#13] [Q&A]
Hi @mustafacagri ,

It would be helpfull if there is info provided how to deploy this app in any of ther free hosting service available like **[Vercel](https://www.netlify.com)** or **[Netlify](https://vercel.com/)**

**Comment by mustafacagri** [comment]
> Hi @mustafacagri ,
> 
> It would be helpfull if there is info provided how to deploy this app in any of ther free hosting service available like **[Vercel](https://www.netlify.com)** or **[Netlify](https://vercel.com/)**

hi @kathirr007, as there are many deployment tutorials for node.js, vue, or nuxt, I did not prioritize this topic. however, if you deployed this repo anywhere, please do not hesitate to share it here; it might be helpful for others, too.
thanks a lot
